modules/users/client/views/verify/verify-account.client.view.html

```diff
@@ -0,0 +1,14 @@
+<section style="margin-top:50px;" class="row text-center" ng-init="vm.validateVerifyToken()">
+
+  <section class="row text-center" ng-if="vm.isResetSent">
+    <h3 class="col-md-12"></h3>
+    <a href="/#!/signin" class="col-md-12">Continue to Sign In</a>
+  </section>
+
+  <section class="row text-center" ng-if="!vm.isResetSent">
```

Comment:
What is "vm.isResetSent"? How does it being truthy tell us that the verification was a success?